Sophmores_FinalProj/Game/Gameplay.cs

```diff
@@ -69,7 +78,16 @@ static void Main(string[] args)
 				//response = UI.PromptInt ("Please select a Door number.");
 			}
 
+		public static Note(string note)
+		{
+			var reader = FIO.OpenReader(FIO.GetLocation("note.txt"), "note.txt");
+			while (!(reader.EndOfStream)) {
 
+				Console.WriteLine(reader.ReadLine());
+			}
+			reader.Close();  
+			return note
```

Comment:
Returned note parameter that you passed in without doing anything to the parameter